ext/enchant/tests/enchant_broker_set_dict_path.phpt

```diff
@@ -0,0 +1,53 @@
+--TEST--
+proto bool enchant_broker_set_dict_path(resource broker, int dict_type, string value) function
+proto string enchant_broker_get_dict_path(resource broker, int dict_type) function
+--CREDITS--
+marcosptf - <marcosptf@yahoo.com.br>
+--SKIPIF--
+<?php
+if(!extension_loaded('enchant')) die('skip, enchant not loader');
+if (!is_resource(enchant_broker_init())) {die("skip, resource dont load\n");}
+if (!is_array(enchant_broker_list_dicts(enchant_broker_init()))) {die("skip, dont has dictionary install in this machine! \n");}
+?>
+--FILE--
+<?php
+$broker = enchant_broker_init();
+$windowsOS = "c:\windows\is\a\bad\OS";
```

Comment:
It's preferable to always escape backslashes, i.e.:

    $windowsOS = "c:\\windows\\is\\a\\bad\\OS";